### DISCUSSION TITLE
How to test if particular notification is being displayed or if it's dismissed

### DISCUSSION THREAD

**Discussion by omnigenous** [dunst-project/dunst#1380] [Q&A]
```sh
notify-send -t 0 "hello"

# do something if notification is closed by user or if it expires in cases when `-t` is not `0`
```

I want to detect when particular notification is dismissed/closed but not sure how.

I tried `--print-id` flag but I don't think that I can use it to kill it.
`$!` also didn't gave me process id to kill

In hopes to check periodically if process is exists

**Comment by bynect** [comment]
you could monitor the notification history. 

#1348 adds something that may be useful for your use case
